core/utils/deprecation.ts

```diff
@@ -22,22 +22,21 @@ goog.declareModuleId('Blockly.utils.deprecation');
 
 /**
  * Warn developers that a function or property is deprecated.
- * @param {string} name The name of the function or property.
- * @param {string} deprecationDate The date of deprecation.
+ * @param name The name of the function or property.
+ * @param deprecationDate The date of deprecation.
  *     Prefer 'month yyyy' or 'quarter yyyy' format.
- * @param {string} deletionDate The date of deletion, in the same format as the
+ * @param deletionDate The date of deletion, in the same format as the
  *     deprecation date.
- * @param {string=} opt_use The name of a function or property to use instead,
- *     if any.
+ * @param opt_use The name of a function or property to use instead, if any.
  * @alias Blockly.utils.deprecation.warn
- * @package
```

Comment:
Replace `package` with `internal` rather than deleting it: https://api-extractor.com/pages/tsdoc/tag_internal/

Reply:
1. Done.
0. Turns out this involves suppressing `nonStandardJsDocs`.  But no problem.
0. Note that the `@package` annotation was deleted by the tooling that converted this file to TS, not by me.  This will be a headache…